### ISSUE TITLE
Can't locate Test/Spelling/Comment.pm

### ISSUE THREAD

**Issue by eserte** [nigelhorne/CGI-Info#8]
It seems that a prereq specification or a skip in the test suite is missing:
```
Can't locate Test/Spelling/Comment.pm in @INC (you may need to install the Test::Spelling::Comment module) (@INC contains: ...  .) at t/comment-spelling.t line 9.
BEGIN failed--compilation aborted at t/comment-spelling.t line 9.
t/comment-spelling.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...

**Comment by nigelhorne** [comment]
Should be fixed in 0.73.

**Comment by nigelhorne** [comment]
Fixed in https://github.com/nigelhorne/CGI-Info/commit/fe45dc67ff2c41c5b1c9c1116eb398c4e2ca3364
